EDU-14302: Managing lazy loading sections on Product Listing Pages (#1723)
vtexdocs/dev-portal-content: docs/faststore/docs/managing-performance/configuring-seo-for-plp.mdx

---
title: "Configuring SEO for Product Listing Pages (PLPs)"
---

> Set up SEO settings for Product Listing Pages (PLPs) to customize title and description templates. This ensures your PLPs align with your brand and improve discoverability. Follow these steps to configure SEO for PLPs:

Reviewer: ```suggestion
Configuring SEO settings for Product Listing Pages (PLPs) is essential for improving search engine visibility and discoverability. This guide walks you through setting up title and description templates for PLPs and provides best practices for optimizing SEO performance.
```

> 

Reviewer: ```suggestion
## Instrucctions

```

1. Open your FastStore project in a code editor of your preference.
2. Go to the `discovey.config.js` file and in the `seo` section add the following:

  ```discovery.config.js
  seo: {
    plp: {
      titleTemplate: '%s | FastStore PLP',
      descriptionTemplate: '%s products on FastStore Product Listing Page',
    },
  },

  ```

  > ⚠ Change the values in `titleTemplate` and `descriptionTemplate` according to your needs. Use `%s` as a placeholder for the PLP title/name.

  - **Title template:** This template will be applied to all PLPs, including collections, categories, subcategories, and brands.
  - **Description template:** This template is used for `collections`, as descriptions for `collections` are not configured in the Catalog. If a specific description isn't configured in the Catalog for a category, subcategory, or brand, this template will be used as a fallback.

> ## Lazy Loading and SEO Considerations

Reviewer: ```suggestion
## Lazy loading and SEO considerations
```
